src/test/python/test_spark.py

```diff
@@ -117,6 +117,7 @@ def test_string_arrays(spark: SparkSession, weaviate_client: weaviate.Client):
     df.write.format("io.weaviate.spark.Weaviate") \
         .option("scheme", "http") \
         .option("host", "localhost:8080") \
+        .option("grpc:enable", "true") \
```

Comment:
Have tested this an works great. I'd prefer a name without the `:` as it is more consistent

Reply:
I was thinking of this `grpc:<options>` approach bc we could then have: `grpc:enable` and `grpc:host` settings (and other if needed) for example the `oidc:<options>` are passed in this manner.
So we have 2 options on how to pass `gRPC setttings`:
1. `grpc:enable` and `grpc:host`
2. `grpcEnable` and `grpcHost`
or we can go with just `grpcHost` setting bc we agreed now that we will have Weaviate running 2 separate ports, wdyt?

Reply:
I have added `grpc:host` and `grpc:secured` options in order to configure `gRPC`